### DISCUSSION TITLE
How to fire mouse:over event when mouse button is down?

### DISCUSSION THREAD

**Discussion by NoftScarlet** [fabricjs/fabric.js#7418] [Q&A]
http://fabricjs.com/hovering
In the example above the mouse:over event isn't triggered if mouse button is held down. Just wondering if there is an option for that?

**Comment by asturur** [comment]
there isn't an option, it could be a bug.
Which button are you holding?

**Reply by NoftScarlet** [reply]
Thanks for the reply. Holding left button down. If it's a bug I'll report as an issue.

**Comment by NoftScarlet** [comment]
It seems mouse:over is blocked by the selection box
[http://jsfiddle.net/ygp9sL24/](http://jsfiddle.net/ygp9sL24/)
In the example above I disabled selection box on canvas and then it works as I wanted. 
Also I would assume adding a bit of offset to selection rectangle and make your mouse cursor "outside" of the selection box can work too.
